api/controllers/main.js

```diff
@@ -1,12 +1,19 @@
 const fs = require('fs');
 const path = require('path');
+const SiteWideErrorLoader = require('../services/SiteWideErrorLoader');
 
 const manifestPath = path.join(__dirname, '..', '..', 'webpack-manifest.json');
 const webpackAssets = JSON.parse(fs.readFileSync(manifestPath, 'utf-8'));
 
 module.exports = {
   index(req, res) {
+    let siteWideError = { display: false };
+    if (req.session.authenticated) {
+      siteWideError = SiteWideErrorLoader.loadSiteWideError();
+    }
+
     res.render('index.html', {
+      siteWideError,
```

Comment:
What about doing the logic for including `siteWideError` here in the controller instead of in the template. That is, something like:

```js
const siteWideError = SiteWideErrorLoader.loadSiteWideError();
const context = { jsBundleName: webpackAssets['main.js'], cssBundleName: webpackAssets['main.css'] };
if (req.isAuthenticated() && siteWideError.display) {
  context.siteWideError = siteWideError;
}
res.render('index.html', context);
```

Then, in `index.html`,  you can just check for presence of `siteWideError`.

This is somewhat stylistic, but I think I prefer it instead of having the mutable `siteWideError` variable created "partially" (ie, just with the `display` property).